acquisitions/src/main/resources/thunderjet/mod-consortia/features/user-tenant-associations.feature

```diff
@@ -486,4 +487,168 @@ Feature: Consortia User Tenant associations api tests
     Then status 200
 
     And match response.totalRecords == 1
-    And match response.permissionUsers[0].permissions == updatedNonEmptyPermissionsOfShadowCentralUser1
\ No newline at end of file
+    And match response.permissionUsers[0].permissions == updatedNonEmptyPermissionsOfShadowCentralUser1
+
+  # We have 'centralUser1' in 'centralTenant' and shadow 'centralUser1' in 'universityTenant'
+  Scenario: If we DELETE real user all records related to this user should be deleted (con-9):
+    # DELETE 'centralUser1'
+    * call read(login) consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Accept': 'text/plain' }
+    Given path 'users', centralUser1.id
+    And headers {'x-okapi-tenant':'#(tenant)', 'x-okapi-token':'#(okapitoken)' }
+    When method DELETE
+    Then status 204
+
+    # 1. verify there is no record in 'users' table in 'central_mod_users' for 'centralUser1'
+    * configure headers = { 'Content-Type': 'application/json', 'Accept': 'application/json' }
```

Comment:
Is test case missed here?

Reply:
This is a functionality of 'mod-users'. Here purpose of this is to indicate we have this case also.

Reply:
okay got it